internal/service/handlers/create_event_type.go

```diff
@@ -44,5 +46,30 @@ func CreateEventType(w http.ResponseWriter, r *http.Request) {
 	}
 	EventTypes(r).Push(typeModel)
 
+	if evtypes.FilterNotOpenable(typeModel) {
```

Comment:
```suggestion
	if typeModel.NoAutoOpen {
```
you'd use filters when you need complex conditions